experimental/Schemes/IdealSheaves.jl

```diff
@@ -698,3 +700,21 @@ end
   end
   return IdealSheaf(X, ID, check=false)
 end
+
+function show_details(I::IdealSheaf)
+  X = scheme(I)
+
+  # If there is a simplified covering, use it!
+  covering = (has_attribute(X, :simplified_covering) ? simplified_covering(X) : default_covering(X))
+  n = npatches(covering)
+  i=1
+  println("Ideal Sheaf on Covered Scheme with ",n," Charts:\n")
+
+
+  ID = IdDict{AbsSpec, Ideal}()
+
+  for U in patches(covering)
+    println("Chart ",i,":\n\t", I(U),"\n")
+    i=i+1
+  end
```

Comment:
You could use `enumerate` to make this more elegant:
```suggestion
  for (i,U) in enumerate(patches(covering))
    println("Chart ",i,":\n\t", I(U),"\n")
  end
```

Reply:
thank you. Changes made accordingly at all occurrences of this PR